frontend/src/app/components/wp-activity/user/user-activity.component.ts

```diff
@@ -172,8 +172,9 @@ export class UserActivityComponent extends WorkPackageCommentFieldHandler implem
         this.wpLinkedActivities.require(this.workPackage, true);
         this.wpCacheService.updateWorkPackage(this.workPackage);
         this.deactivate(true);
+        this.inFlight = false;
       })
-      .catch(() => this.deactivate(true));
+      .catch(() => {this.deactivate(true); this.inFlight = false; });
```

Comment:
Instead of adding 
`this.inFlight = false` 
twice, please use `finally` to keep the code dry. I also think that it would read better.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/finally